lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -81,6 +81,7 @@ def return_csv(self, filename, header, data):
         writer = csv.writer(csv_file, dialect='excel', quotechar='"',
                             quoting=csv.QUOTE_ALL)
 
+        header = [x.encode('utf-8') for x in header]
```

Comment:
I think this change and https://github.com/eduNEXT/edx-platform/blob/542eb7647866e72d817cda13470782655fe9e835/lms/djangoapps/dashboard/sysadmin.py#L316 are not needed but I'm not 100% sure on how to test that